### PR TITLE
Feature/134 navigation header

### PR DESCRIPTION
**Pull request by ahoseinian**
### using react-navigation header sample.

this is a sample one-page change to react-navigation header instead of the ui-kitten version.
- I think the way react-navigation works is a bit more powerful and allow to change the header depending on which stack a route is on
- safe area view problem would be easier solved https://github.com/litentry/litentry-app/pull/111#discussion_r652590811
- component only handles the content 

on the downside
- component and header are separated so you have to look for it.

### CODE REVIEW

**Comment by ahoseinian** [comment]
@franjs @geastwood I just changed one screen if you approve it will do the rest

**Review by silva-fj — DISMISSED** [review]
It looks good 👍🏼

File: src/styles.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ const globalStyles = StyleSheet.create({
     flex: 1,
   },
   icon: {...inlineIconDimension, ...iconColor},
+  icon30: {width: 25, height: 25},
```
Comment (silva-fj) [inline src/styles.ts]: What does `icon30` mean?

File: src/navigation/routeKeys.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
 export const dashboard = 'Dashboard' as const;
-export const tips = 'Tips' as const;
-export const tipDetail = 'TipDetail' as const;
-export const submitTip = 'SubmitTipScreen' as const;
-export const registrarList = 'RegistrarList' as const;
-export const motionDetail = 'MotionDetail' as const;
-export const notificationSettingsScreen = 'NotificationSettingsScreen' as const;
+export const tipsScreen = 'Tips' as const;
```
Comment (silva-fj) [inline src/navigation/routeKeys.ts:2]: NIT: Why only adding "screen" to this one? I think we should either drop or add "Screen" to all of them

File: src/navigation/routeKeys.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+export const tipDetail = 'Tip' as const;
+export const submitTip = 'SubmitTip' as const;
+export const registrarList = 'Registrars' as const;
+export const motionDetail = 'Motion' as const;
+export const notificationSettingsScreen = 'Notification' as const;
```
Comment (silva-fj) [inline src/navigation/routeKeys.ts:7]:
```suggestion
export const notificationSettingsScreen = 'NotificationSettings' as const;
```

**Review by silva-fj — DISMISSED** [review]

File: src/styles.ts (hunk 1 above)
Comment (ahoseinian) [reply src/styles.ts]: changed to `icon25` it was a mistake.

File: src/navigation/routeKeys.ts (hunk 2 above)
Comment (ahoseinian) [reply src/navigation/routeKeys.ts:2]: I added `Screen` to all 👍

**Review by geastwood — APPROVED** [review]
Godspeed

**Pull request merged**